integration/mocha-tests/utils/HelperFunctions.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+const { spawnSync, spawn } = require('child_process');
+
+// Run the command asynchronously without blocking the Node.js event loop.
+function runAsync(command, args, options) {
```

Comment:
Feel like this should not be in this test framework as it is just a script helper.